### DISCUSSION TITLE
Is there a script available to convert PPOCRLabel KIE dataset format to the same format as wildreceipt format

### DISCUSSION THREAD

**Discussion by bpm-ui** [PaddlePaddle/PaddleOCR#7392] [Q&A]
I'm trying to set up my own dataset using PPOCRLabel using KIE mode. I was able to run the gen_ocr_train_val_test.py script after the dataset was created, but the output is not in the same format as the https://paddleocr.bj.bcebos.com/ppstructure/dataset/wildreceipt.tar dataset sample.

Is there a script available to convert PPOCRLabel KIE output, or is the class_list.txt, train.txt and test.txt manually created from the PPOCRLable output?

**Comment by littletomatodonkey** [comment]
Hi, the file format you export using PPOCRLabel can be directly used for the KIE model training, which is same as wildreceipt format we provide, the only thing you need to do is preparing for the class_list.txt, you can refer to this [tut](https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.6/doc/doc_en/kie_en.md#12-custom-dataset) for more details.

**Comment by MohamedElghazi01** [comment]
Hi dear Do you find the answer , if not respond to give you the script to convert label.txt to train.txt from wildreceipt format
